Sources/TuistCore/Models/Dependency.swift

```diff
@@ -6,13 +6,27 @@ public enum SDKStatus {
     case optional
 }
 
+public enum SDKSource {
+    case developer
+    case system
+
+    public var path: String {
```

Comment:
The logic that determines the path currently lives in `SDKNode.init`, perhaps we can create a helper function there to determine the path taking into account the platform as well (which may not be iOS).

Reply:
Agreed